### ISSUE TITLE
Repository name and documentation

### ISSUE THREAD

**Issue by spMohanty** [geotagx/geotagx#1]
I am not sure, if its the appropriate name for the repository, as it is hogging the root name of the organization, and most visitors will assume this is the core repository of the whole project.

Also, some documentation about what this does will be really helpful

**Comment by elliec** [comment]
yes I really agree, same goes for whatever else that we put up there. I was looking through the repository today and most things seem to be missing documentation. We need to fix this asap.

regarding the name it should be more explicit (eg i thought it was the theme). I really don't like having a folder with the same name as it's parent it makes navigation difficult

**Comment by supranove** [comment]
This repo is meant to include the entire GeoTag-X platform plus some scripts and configurations that make creating a GeoTag-X instance a breeze on any server. Since it's not clear what it does yet, how about I transfer ownership of the repo to myself, flesh it out a little, and then if there's a need for it in the future, re-transfer ownership back to geotagx? Thoughts @elliec, @spMohanty?
